### DISCUSSION TITLE
Add a new object to the previous model but failed

### DISCUSSION THREAD

**Discussion by MIIXNON** [ultralytics/yolov5#8541] [Q&A]
I already have a cat&dog classification model, and then I added a new object to detect if someone is smoking in the picture, I used the best.pd from the cat&dog classifier to train the cigarette dataset alone, but the results show that the resulting model only recognises smoking, but not the previous cat&dog picture. I wonder if I have to combine the smoking dataset with the original dataset and retrain it again? Although this is a bit costly

**Comment by glenn-jocher** [comment]
@MIIXNON see https://community.ultralytics.com/t/how-to-combine-weights-to-detect-from-multiple-datasets

**Reply by MIIXNON** [reply]
Really appreciated. This helps a lot!
